<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Leg Asymmetry Analysis Tool</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_Manage">
     <attribute name="title">
      <string>Manage</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Add">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>0</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Delete">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>480</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Delete File</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_Manager">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>631</width>
        <height>511</height>
       </rect>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="columnCount">
       <number>1</number>
      </property>
      <column/>
     </widget>
     <widget class="QPushButton" name="pushButton_Left">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>240</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load Left</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Right">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>280</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Load Right</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Link">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>120</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Link Sets</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Unlink">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>160</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Unlink Sets</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget_Backup" native="true">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>360</y>
        <width>121</width>
        <height>71</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_Server">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Toggle Backup</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Local">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>121</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Toggle Local</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_Filter">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>40</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Toggle Filter</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Analysis">
     <attribute name="title">
      <string>Analysis</string>
     </attribute>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>781</width>
        <height>561</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_Dataset">
       <attribute name="title">
        <string>Dataset</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_Left">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>381</width>
          <height>451</height>
         </rect>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>20</number>
        </property>
        <property name="columnCount">
         <number>10</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
       <widget class="QTableWidget" name="tableWidget_Right">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>30</y>
          <width>381</width>
          <height>451</height>
         </rect>
        </property>
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="rowCount">
         <number>20</number>
        </property>
        <property name="columnCount">
         <number>10</number>
        </property>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>68</number>
        </attribute>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <row/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
        <column/>
       </widget>
       <widget class="QLabel" name="label_Right">
        <property name="geometry">
         <rect>
          <x>390</x>
          <y>0</y>
          <width>381</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Right Leg</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_Left">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>381</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Left Leg</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ClearLeft">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>490</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Clear Left</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ClearRight">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>490</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Clear Right</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_ClearBoth">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>330</x>
          <y>490</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Clear Both</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_Results">
       <attribute name="title">
        <string>Graphs</string>
       </attribute>
       <widget class="QFrame" name="frame">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>250</y>
          <width>120</width>
          <height>80</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QTabWidget" name="tabWidget_Graphs">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>751</width>
          <height>521</height>
         </rect>
        </property>
        <property name="tabPosition">
         <enum>QTabWidget::East</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Graph 1</string>
         </attribute>
         <widget class="QWidget" name="widget_Graph1" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>721</width>
            <height>511</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Graph 2</string>
         </attribute>
         <widget class="QWidget" name="widget_Graph2" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>721</width>
            <height>511</height>
           </rect>
          </property>
         </widget>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_Model">
       <attribute name="title">
        <string>Results</string>
       </attribute>
       <widget class="QTextBrowser" name="textBrowser_Results">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>751</width>
          <height>511</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
         </font>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_Analyze">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>674</x>
        <y>0</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Analyze</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_Analysis">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>471</width>
        <height>16</height>
       </rect>
      </property>
      <property name="value">
       <number>0</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Settings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Connect">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Creds">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_Creds">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>210</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Remember Credentials</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Creds">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>170</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Creds">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>150</y>
        <width>131</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Credential File</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
